### ISSUE TITLE
Parsing elevatedButton with the elevation property 

### ISSUE THREAD

**Issue by acx70** [peiffer-innovations/json_theme#28]
I get an error when I try to parse an **elevatedButton** with a **double** value in the property **elevation**. It works instead if a pass a **string.**

Working code:

```
}
......
"elevatedButtonTheme": {
    "style": {
      "backgroundColor": "#068a9c",
      "side": {
        "color": "#068a9c",
        "width": 2.0
      },
      "shape": {
        "type": "rectangle",
        "borderRadius": 20.0
      },
      "elevation": "20.0"
    }
  }
......
}
```

Not working code:

```
}
......
"elevatedButtonTheme": {
    "style": {
      "backgroundColor": "#068a9c",
      "side": {
        "color": "#068a9c",
        "width": 2.0
      },
      "shape": {
        "type": "rectangle",
        "borderRadius": 20.0
      },
      "elevation": 20.0
    }
  }
......
}
```

**Comment by jpeiffer** [comment]
Should be fixed in 3.1.2+1.

**Comment by acx70** [comment]
It works with ^3.1.2+1
